docs/guide/counter/plugins/counter/counter_test.go

```diff
@@ -19,14 +22,18 @@ func TestCounterPlugin(t *testing.T) {
 	// Basecoin initialization
 	eyesCli := eyescli.NewLocalClient("", 0)
 	chainID := "test_chain_id"
-	bcApp := app.NewBasecoin(eyesCli)
+
+	// l := log.TestingLogger().With("module", "app"),
+	l := log.NewTMLogger(os.Stdout).With("module", "app")
```

Comment:
log is taken by the package.  you can use logger. i like short names.  but will abide by the coding standard.